rest/queue/list-post-example-1/list-post-example-1.5.x.py

```diff
@@ -3,8 +3,10 @@
 
 # Your Account Sid and Auth Token from twilio.com/user/account
 account_sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
-auth_token  = "your_auth_token"
+auth_token = "your_auth_token"
 client = TwilioRestClient(account_sid, auth_token)
 
-queue = client.queues.create("newqueue", )
-print(queue.sid)
\ No newline at end of file
+queue = client.queues.create(
+    "newqueue",
```

Comment:
Good catch! actually, that can be the reason! I'll try some changes and see if the formatter does the same thing.